src/PeopleContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useState } from 'react';
+import { Person } from './types';
+
+type ContextProps = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.